package.json

```diff
@@ -12,8 +12,7 @@
     "md:lint": "npx markdownlint-cli '*.md' 'docs/*.md'",
     "md:spell": "npx markdown-spellcheck -a -n -x --en-us '*.md' 'docs/*.md'",
     "test": "echo \"Error: testing disabled\" && exit 0",
-    "package": "web-ext build",
```

Comment:
I think you might need to update the README doc, since I think it refers to the `package` script in a couple places.